modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheOnCopyFlagLocalSelfTest.java

```diff
@@ -19,13 +19,16 @@
 
 import org.apache.ignite.cache.CacheAtomicityMode;
 import org.apache.ignite.cache.CacheMode;
+import org.junit.Ignore;
 
 import static org.apache.ignite.cache.CacheAtomicityMode.ATOMIC;
 import static org.apache.ignite.cache.CacheMode.LOCAL;
 
 /**
  * Tests {@link org.apache.ignite.cache.CacheInterceptor}.
  */
+// GG-11141.
```

Comment:
Here in the next 4 files: we do not need this comment since we do not have such ticket in Apache Ignite trackers. IGNITE-9214 is enough